frontend/screens/inbox/InboxInterface.js

```diff
@@ -25,90 +39,92 @@ const InboxInterface = () => {
       console.log(_influencer, _brand)
       if (_influencer) {
         id = _influencer;
-        data = await getAllConversations(id, 'influencer',setConversations,showAlert)
+        data = await getAllConversations(id, 'influencer', setConversations, showAlert)
         setUserId(id)
         setUserType('influencer')
       } else if (_brand) {
         id = _brand;
-        data = await getAllConversations(id, 'brand',setConversations,showAlert)
+        data = await getAllConversations(id, 'brand', setConversations, showAlert)
         setUserId(id)
         setUserType('brand')
       }
     }
-    getData(); 
+    getData();
   }, []);
 
   const handleSearchChange = (text) => {
     setSearchQuery(text);
   };
   return (
-    <ScrollView contentContainerStyle={styles.scrollViewContent}>
-      <View style={[styles.inboxContainer, { height: Dimensions.get("window").height }]}>
-        <View style={styles.headerContainer}>
-          <View style={styles.headerContent}>
-            <TouchableOpacity onPress={() => navigation.navigate("UserProfile")}>
-              <View style={styles.headerTitleContainer}>
-                <Text style={styles.headerTitle}>Inbox</Text>
-              </View>
-            </TouchableOpacity>
-            <TouchableOpacity onPress={() => navigation.navigate("NewMessage")}>
-              <View style={styles.newMessageButton}>
-                <Image
-                  style={styles.newMessageIcon}
-                  contentFit="cover"
-                  source={require("../../assets/depth-5-frame-021.png")}
-                />
-              </View>
-            </TouchableOpacity>
-          </View>
+    <View style={{ width: "100%", height: "100%" }}>
+      <View style={styles.headerContainer}>
+        <View style={styles.headerContent}>
+          <TouchableOpacity onPress={() => navigation.navigate("UserProfile")}>
+            <View style={styles.headerTitleContainer}>
+              <Text style={styles.headerTitle}>Inbox</Text>
+            </View>
+          </TouchableOpacity>
+          <TouchableOpacity onPress={() => navigation.navigate("NewMessage")}>
+            <View style={styles.newMessageButton}>
+              <Image
+                style={styles.newMessageIcon}
+                contentFit="cover"
+                source={require("../../assets/depth-5-frame-021.png")}
```

Comment:
the name of the file isn't very informative, can you rename the file so that ut clearly states what's inside it?